src/app/server.ts

```diff
@@ -0,0 +1,19 @@
+import express from 'express';
+import { rollDice } from '../diceRoll';
+const app = express();
+const port = 3000;
+
+app.get('/', (req, res) => {
+  res.send('Hola don Pepe!');
+});
+
+app.get('/dice/:faces', (req, res) => {
+  const faces = req.params.faces;
+  const result = rollDice(faces);
+  res.json({ result: result });
```

Comment:
Podrías hacerlo todo directo

```suggestion
app.get('/dice/:faces', (req, res) => {
  res.json({ result: rollDice(req.params.faces) });
```

Reply:
Por ponernos tiquismiquis